src/main/java/seedu/address/model/insurancepolicy/InsurancePolicy.java

```diff
@@ -4,18 +4,25 @@
 
 import java.util.Objects;
 import java.util.Optional;
+import java.util.regex.Pattern;
 
 /**
  * Represents an InsurancePolicy in the address book.
  * Guarantees: immutable;
  */
 public class InsurancePolicy {
 
-    public static final String MESSAGE_CONSTRAINTS = "PolicyIDs should not contain '>'!. URLs should be "
-            + "preceded by '>' after the PolicyID.";
+    public static final String MESSAGE_CONSTRAINTS = "Policy input is of an incorrect format."
+            + "\nPolicyIDs should not contain '>'."
+            + "\nOptional URLs should be preceded by '>' after the PolicyID, and should be of a standard URL format."
+            + "\nDo head over to our User Guide if you are unsure of the command or input format for policies.";
     public static final String MESSAGE_NO_POLICY = "Currently does not have an active policy";
+    public static final Pattern URL_REGEX = Pattern.compile("((ftp|http|https):\\/\\/)?(www.)?(?!.*(ftp|http"
+            + "|https|www.))[a-zA-Z0-9_-]+(\\.[a-zA-Z]+)+((\\/)[\\w#]+)*(\\/\\w+\\?[a-zA-Z0-9_]+=\\"
+            + "w+(&[a-zA-Z0-9_]+=\\w+)*)?", Pattern.CASE_INSENSITIVE);
```

Comment:
Could you add a comment to explain what this regex checks for? Just for readability, since its quite hard to understand.

Reply:
Added some comments for this and another regular expression that I used.